Client/ClientGame.js

```diff
@@ -242,10 +244,10 @@ Game.prototype.attack = function(data){
 	for (var i=0; i<GameObjManager.GameObjectList.length; i++){
 		var temp = GameObjManager.GameObjectList[i];
 		if((data.id!=this.localPlayerid)&&(temp.id == data.id)){
-			var item = new Item(data.pos.x + 60*(data.direction),data.pos.y,10,10, data.itemId, true);
+			var item = new Item(data.pos.x + 60*(data.x_direction),data.pos.y,10,10, data.itemId, true);
```

Comment:
60*(data.x_direction)
This rotates the item around the players body to shoot it in the right direction correct?